src/db/models/project.ts

```diff
@@ -0,0 +1,12 @@
+import * as t from 'io-ts';
+
+import { brandedType } from '../../util/io-ts';
+import type { Brand } from '../../util/types';
+
+export type ProjectId = Brand<
+  number,
+  { readonly s: unique symbol },
+  'project.id'
+>;
+
+export const PROJECT_ID = brandedType<number, ProjectId>(t.number);
```

Comment:
Why didn't we added rest of the fields in this model ?

Reply:
Because I'm taking minimalistic approach here :)

I only need `projectVersion` and `projectVersionPlan` defined, but need these ID properties defined in order to reference them. The rest will be defined when there's a need. It's a usual approach we take in this repo, look at `attachmentPrototype.ts`